### PR TITLE
lease: modify log info

### PR DESCRIPTION
**Pull request by CabinfeverB**
### What problem does this PR solve?

Issue Number: ref #7700

### What is changed and how does it work?
modify log message add some fields.

```commit-message
```

### Check List

Tests

- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

File: pkg/election/lease.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func (l *lease) KeepAlive(ctx context.Context) {
 			// https://pkg.go.dev/time@master#Timer.Reset
 			timer.Reset(l.leaseTimeout)
 		case <-timer.C:
-			log.Info("lease timeout", zap.Time("expire", l.expireTime.Load().(time.Time)), zap.String("purpose", l.Purpose))
+			log.Info("keep alive lease too slow", zap.Duration("timeout duration", l.leaseTimeout), zap.Time("actual-expire", l.expireTime.Load().(time.Time)), zap.String("purpose", l.Purpose))
```
Comment (HuSharp) [inline pkg/election/lease.go:138]:
```suggestion
			log.Info("keep alive lease too slow", zap.Duration("timeout-duration", l.leaseTimeout), zap.Time("actual-expire", l.expireTime.Load().(time.Time)), zap.String("purpose", l.Purpose))
```

**Review by HuSharp — APPROVED** [review]

File: pkg/election/lease.go [hunk 2]
```diff
@@ -154,11 +154,14 @@ func (l *lease) keepAliveWorker(ctx context.Context, interval time.Duration) <-c
 
 		log.Info("start lease keep alive worker", zap.Duration("interval", interval), zap.String("purpose", l.Purpose))
 		defer log.Info("stop lease keep alive worker", zap.String("purpose", l.Purpose))
-
+		lastTime := time.Now()
 		for {
-			go func() {
+			start := time.Now()
+			if start.Sub(lastTime) > interval*2 {
+				log.Warn("the interval between keeping alive lease is too long", zap.Time("last-time", lastTime))
```
Comment (rleungx) [inline pkg/election/lease.go:161]: Will it print a lot?

Comment (CabinfeverB) [reply pkg/election/lease.go:161]: It's possible. But similar to the 'clock offset' warning log in tso, it is indeed a worthwhile log to output

**Review by rleungx — APPROVED** [review]

**Pull request merged**